src/components/Product/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./style.css";
+
+const Product = ({ image, name, description, price }) => {
+  return (
+    <div className="product">
+      <img className="product__image" src={`assets/${image}`} />
```

Comment:
hele normálně to funguje (reaguju na tvojí otázku z moje.czechitas), protože tomu předáváš "image"
```
       <Product
          image="red-shoes.jpg"
```
a pak se z toho stane 
`      <img className="product__image" src="assets/red-shoes.jpg" />`
a to je v pořádku